devops/openshift/workers-deploy.yml

```diff
@@ -65,6 +65,21 @@ objects:
                     secretKeyRef:
                       key: ${ZEEBE_AUTHORIZATION_SERVER_URL_KEY}
                       name: ${WORKERS_SECRET_NAME}
+                - name: GC_NOTIFY_URL
```

Comment:
As workers does not talk to GC notify API, these properties are not required to be added.

May be the property GC_NOTIFY_TO_ADDRESS_NAME_KEY is required in future when workers are sending an email to ministry inbox. But not now.(@andrewsignori-aot  mentioned this in one of our discussions)

Reply:
yes you are right @dheepak-aot